core/src/com/unciv/ui/overviewscreen/GlobalPoliticsOverviewTable.kt

```diff
@@ -125,7 +125,7 @@ class GlobalPoliticsOverviewTable (
 
     private fun getCivInfoTable(civ: CivilizationInfo): Table {
         val civInfoTable = Table(skin)
-        val leaderName = civ.getLeaderDisplayName().removeSuffix(" of " + civ.civName)
+        val leaderName = civ.nation.leaderName
```

Comment:
.tr(), this looks like it's currently not translated at all?

Reply:
the line just below this one adds `leaderName.toLabel()` which should translate the string